### PR TITLE
Add links to linked to/from to curation pages (#1428)

### LINKED ISSUE #1428: On curation pages, add pages linked to/from target item
as per https://twitter.com/egonwillighagen/status/1387489358136201219

### PR DESCRIPTION
**Pull request by carlinmack**
Close #1428 

image

### CODE REVIEW

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -59,8 +59,9 @@
      
     $.getJSON(url, function (data) {
 	 var item = data.entities["{{ q }}"];
+   var title = item.labels.en.value
```
Comment (egonw) [inline scholia/app/templates/base.html:62]: This does not look good: the `if` in the next line is there before `item.labels.en` can be non-existing.

**Review by egonw — COMMENTED** [review]
Can you check out my comment?

File: scholia/app/templates/base.html (hunk 1 above)
Comment (carlinmack) [reply scholia/app/templates/base.html:62]: oh yep of course

**Review by Daniel-Mietchen — APPROVED** [review]
Looks good and works fine.

File: scholia/app/templates/base.html [hunk 2]
```diff
@@ -60,7 +60,19 @@
...
 	 var item = data.entities["{{ q }}"];
 	 if ('en' in item.labels) {
-	     $("#h1").text(item.labels.en.value);
+	     var title = item.labels.en.value
+	     $("#h1").text(title);
+
+	     {% if request.path.endswith("curation") or request.path.endswith("curation/") %}
+
+	     var linkedToFrom = "<p><a href='https://www.wikidata.org/w/index.php?hidecategorization=1&target={{ q }}&showlinkedto=1&limit=500&days=7&enhanced=1&title=Special:RecentChangesLinked&urlversion=2' >";
+	     linkedToFrom += "Recent changes to items that link <em>to</em> " + title + "</a> and ";
```
Comment (fnielsen) [inline scholia/app/templates/base.html:69]: Shouldn't `title` be escaped, e.g., an article title could have a "<" in it? (and XKCD 327)

File: scholia/app/templates/base.html [hunk 3]
```diff
@@ -60,7 +60,19 @@
...
+	     linkedToFrom += "<a href='https://www.wikidata.org/w/index.php?hidecategorization=1&target={{ q }}&showlinkedto=0&limit=500&days=7&enhanced=1&title=Special:RecentChangesLinked&urlversion=2' >";
+	     linkedToFrom += "recent changes to items that link <em>from</em> " + title + "</a>.</p>";
```
Comment (fnielsen) [inline scholia/app/templates/base.html:71]: Also here?

File: scholia/app/templates/base.html [hunk 4]
```diff
@@ -60,7 +60,19 @@
     $.getJSON(url, function (data) {
 	 var item = data.entities["{{ q }}"];
 	 if ('en' in item.labels) {
```
Comment (Daniel-Mietchen) [inline scholia/app/templates/base.html:62]: perhaps this should be making use of ```entityToLabel``` from scholia.js.

Comment (fnielsen) [reply scholia/app/templates/base.html:62]: Isn't that outside the PR?

File: scholia/app/templates/base.html [hunk 5]
```diff
@@ -60,7 +60,19 @@
...
-	     $("#h1").text(item.labels.en.value);
+	     var title = item.labels.en.value
+	     $("#h1").text(title);
+
+	     {% if request.path.endswith("curation") or request.path.endswith("curation/") %}
+
+	     var linkedToFrom = "<p><a href='https://www.wikidata.org/w/index.php?hidecategorization=1&target={{ q }}&showlinkedto=1&limit=500&days=7&enhanced=1&title=Special:RecentChangesLinked&urlversion=2' >";
+	     linkedToFrom += "Recent changes to items that link <em>to</em> " + $('<div>').text(title).html() + "</a> and ";
+	     linkedToFrom += "<a href='https://www.wikidata.org/w/index.php?hidecategorization=1&target={{ q }}&showlinkedto=0&limit=500&days=7&enhanced=1&title=Special:RecentChangesLinked&urlversion=2' >";
+	     linkedToFrom += "recent changes to items that link <em>from</em> " + $('<div>').text(title).html() + "</a>.</p>";
```
Comment (fnielsen) [inline scholia/app/templates/base.html:71]: escapeHTML is defined in Scholia.js

Comment (carlinmack) [reply scholia/app/templates/base.html:71]: Ah yes that would be better to use

**Pull request merged**